cmd/algod/main.go

```diff
@@ -232,10 +232,22 @@ func run() int {
 					telemetryConfig.SessionGUID = *sessionGUID
 				}
 			}
-			err = log.EnableTelemetry(telemetryConfig)
```

Comment:
@urtho Lets add this back, and just kick off the async loop only in the error case.